javascript/organizing_your_javascript_code/webpack.md

```diff
@@ -42,6 +42,13 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 // ...
 ```
 
+<div class="lesson-note lesson-note--tip" markdown="1">
+ <h4>A note if your fonts are not loading</h4>
+ If you have any issues loading fonts with webpack, the issue is not with webpack, but how you declare the css rule.
+
+ Check out <a href="https://discord.com/channels/505093832157691914/505093832157691916/1138881744850931835" target="_blank >this </a>answer from Stack Overflow if you get stuck.
```

Comment:
Can we link to the SO thread directly here instead please? Not all our users are on Discord and won't be able to access the Discord links.

Reply:
Absolutely, that's my mistake, I will update the link

Reply:
That's a good point. I'm just trying to figure out how to amend the PR and I will make both of those changes, thanks for pointing it out